ext/json/php_json_encoder.h

```diff
@@ -26,6 +26,7 @@ struct _php_json_encoder {
 	int depth;
 	int max_depth;
 	php_json_error_code error_code;
+	smart_str *indent_str;
```

Comment:
This should store zend_string* rather than smart_str*.

Reply:
Done!